mne_qt_browser/conftest.py

```diff
@@ -6,9 +6,20 @@
 
 import pytest
 
+from mne.viz import use_browser_backend
+from mne.conftest import *  # noqa: F401, F403
```

Comment:
star imports are best avoided, can't you just import `garbage_collect` here?

Reply:
I see now this is a remnant of the old "copy-tests"-CI approach which is now deprecated since you made the CI-overhaul. Thanks